src/js/components/PreSubmitSection.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import ErrorableCheckbox from './ErrorableCheckbox';
+
+// formConfig.preSubmitInfo = {
```

Comment:
It might be useful to add some comments about how `preSubmitInfo` comes from the form config, and how the other props (`onChange`, `checked`, `showError`) get passed, to distinguish them from one another.